admin/services/pupil.service.js

```diff
@@ -25,6 +25,29 @@ const pupilService = {
     }
   },
   /**
+   * Returns pupils filtered by school and sorted by field and direction (asc/desc)
+   * @param schoolId
+   * @param sortingField
+   * @param sortingDirection
+   * @returns {Promise.<*>}
+   */
+  fetchSortedPupilsData: async (schoolId, sortingField, sortingDirection) => {
+    // TODO: Introduce integration tests
+    try {
+      const sort = {}
+      sort[sortingField] = sortingDirection
+      return await Pupil
+        .find({
+          'school': schoolId
+        })
+        .sort(sort)
+        .lean()
+        .exec()
+    } catch (error) {
+      throw new Error(error)
```

Comment:
Is there actually any point to throwing this error?  Why catch this error in the first place if we are not handling it?

Reply:
Well not for now if not handled, in fact I usually add a console.log because it's useful.

Reply:
Then I suggest just removing the try/catch.  Just let the caller catch them.

Reply:
IT's been removed in the next PR.